src/components/AdminPortal/EventRegistrantsModal/Modal/EventRegistrantsModal.spec.tsx

```
@@ -686,7 +897,13 @@ describe('EventRegistrantsModal', () => {
       const keyPress = key.length === 1 ? key : `{${key}}`;
       await user.keyboard(keyPress);
 
+      // Use queryBy for negative assertion
       expect(screen.queryByTestId('add-onspot-modal')).not.toBeInTheDocument();
+
+      // Small delay between iterations
+      await waitFor(() => {
+        expect(screen.queryByTestId('add-onspot-modal')).not.toBeInTheDocument();
```

ESLint

[error] Replace `screen.queryByTestId('add-onspot-modal')` with `⏎··········screen.queryByTestId('add-onspot-modal'),⏎········`

(prettier/prettier)